startGame.js

```diff
@@ -0,0 +1,68 @@
+//variable which will allow us to insert out timer into right div
+var startTimer = document.getElementById("timer")
+
+//this element connects to the start button
+$("#startbutton").click( function(){
+    createTimer()
+    hideStartBtn()
+    displayplayScreen()
+    hideGameTitle()
+    displayGameBox()
+    hideTopSection()
+})
+
+//this function hides the start button after it has been clicked
+function hideStartBtn() {
+        document.getElementById("startbutton").style.display = "none"
+}
+
+// function to make the header disappear
+function hideGameTitle() {
+    document.getElementById("title").style.display = "none"
```

Comment:
this is called aboveBox now, not title

Reply:
On line 5 you need to add psr2 so a space between the curly bracket and the other bracket ) {